### ISSUE TITLE
Doing OPC for SKY130 process node?

### ISSUE THREAD

**Issue by mithro** [vincentlv/DimmiLitho#6]
Hi!

About two years ago, @Google partnered with SkyWater to release a fully open source PDK for SkyWater's 130nm process technology (https://skywater-pdk.rtfd.io https://github.com/google/skywater-pdk). With silicon now starting to come back from the open source MPW program, there is a whole bunch of imaging happening which compares the generated GDS to the actual structures found in silicon. As well, I'm working on publishing publically imaging of the masks that were used to manufacture the open source MPW program. Lastly, Google released an optimized SRAM build space (https://github.com/google/skywater-pdk-libs-sky130_fd_bd_sram) which includes precalculated OPC.

As well, last week Google announced it has received funding to expand the program to include a 90nm process technology - https://opensource.googleblog.com/2022/07/SkyWater-and-Google-expand-open-source-program-to-new-90nm-technology.html

With all this, it would be awesome to have a fully open source OPC tooling that was compatible with these process technologies. 
It would also be awesome to see examples in this repository targeting these real world, manufacturable processes.

**Comment by mithro** [comment]
I believe that @mguthaus who leads the OpenRAM project has done some early exploration -- see his twitter posts @ https://twitter.com/mguthaus/status/1467192007458103298?s=20

**Comment by mithro** [comment]
If you are interested in discussing what could be done here, I suggest joining the https://open-source-silicon.dev/ slack space or emailing me at tansell@google.com

**Comment by mguthaus** [comment]
Definitely include me as well. I'm very interested in this.

**Comment by vincentlv** [comment]
Hi mithro. Very good to know that. While I just feel not sure if there is a real need for a real OPC. Usually, designers just design patterns in final wafer and are transparent to how foundry fabricate it through OPC. Almost, fab guy take responsibility to design the mask, because the mask could be quite different from different fabs, process recipes, or different litho conditions, which only fab know.

**Comment by mithro** [comment]
Hi @vincentlv,

We (Google) are working closely with the SkyWater foundry and looking at deploying new OPC tooling in mask generation to push the boundary of things like SRAM and flash densities on these older process nodes. We are also exploring non-traditional mask structures which existing OPC technologies struggle to analyze. Hence, why we have interest in open source tooling in this space (which we can also help improve). 

We have also been talking with tooling companies like ASML about creating open models of their older machines.

Basically, we want to get more performance out of these older technologies which are extremely cheap to work with and better OPC is a clear area that can help.

Does that help explain things?

**Comment by vincentlv** [comment]
Yes, that explains well. Now I see the point.

**Comment by mithro** [comment]
Plus, all this can be publically accessible for anyone to look at and understand!
